spec/lib/smb2/packet/close_request_spec.rb

```diff
@@ -1,5 +1,5 @@
+require 'spec_helper'
```

Comment:
If you're using rspec >= 3 you can do the following in `.rspec`:

```
--require spec_helper
```

and never have to `require 'spec_helper'` at the top of your specs again.  I've been doing that as I convert metasploit-* projects to rspec 3.1/3.2.